sorc/orog_mask_tools.fd/lake.fd/lakefrac.F90

```diff
@@ -39,34 +39,42 @@ PROGRAM lake_frac
     REAL, PARAMETER :: d2r = acos(-1.0) / 180.0
     REAL, PARAMETER :: r2d = 180.0 /acos(-1.0) 
     REAL, PARAMETER :: pi = acos(-1.0) 
-    REAL*8, PARAMETER :: gppdeg = 119.99444445
-    REAL*8, PARAMETER :: delta = 1.0 / 119.99444445
+    REAL*8, PARAMETER :: gppdeg = 120.0
+    REAL*8, PARAMETER :: delta = 1.0 / 120.0
 
-    CHARACTER(len=32) :: arg
+    CHARACTER(len=32) :: arg, lakestatus_srce, lakedepth_srce
     CHARACTER(len=256) :: lakedata_path
     INTEGER :: stat
     
     CALL getarg(0, arg) ! get the program name
-    IF (iargc() /= 3 .AND. iargc() /= 4) THEN
+    IF (iargc() /= 5 .AND. iargc() /= 6) THEN
       PRINT*, 'Usage: ', trim(arg), & 
-       ' [tile_num (0:all tiles, 7:regional)] [resolution (48,96, ...)] [path to lake data file]'  
+       ' [tile_num (0:all tiles, 7:regional)] [resolution (48,96, ...)] &
+         [lake data path] [lake status source] [lake depth source]'  
       PRINT*, 'Or: ', trim(arg), & 
-       ' [tile_num (0:all tiles, 7:regional)] [resolution (48,96, ...)] [path to lake data file] [lake_cutoff]'  
-      STOP
+       ' [tile_num (0:all tiles, 7:regional)] [resolution (48,96, ...)] &
+         [lake data path] [lake status source] [lake depth source] [lake_cutoff]'
+      STOP -1
     ENDIF
     CALL getarg(1, arg)
     READ(arg,*,iostat=stat) tile_req
     CALL getarg(2, arg)
     READ(arg,*,iostat=stat) cs_res
     CALL getarg(3, lakedata_path)
+    CALL getarg(4, lakestatus_srce)
+    CALL getarg(5, lakedepth_srce)
 
-    IF (iargc() == 3) THEN
+    IF (iargc() == 5) THEN
```

Comment:
add line after lake_cutoff line:
gridcell_lake = 1

Reply:
it seems like there is some inconsistency on default values; in some places lake_cutoff = 0.5, in others, lake_cutoff = 0.2; same for binary_lake, in some places it's 1 in others 0